core/node/api_server/src/tx_sender/master_pool_sink.rs

```diff
@@ -33,6 +38,10 @@ impl TxSink for MasterPoolSink {
     ) -> Result<L2TxSubmissionResult, SubmitTxError> {
         let address_and_nonce = (tx.initiator_account(), tx.nonce());
 
+        if self.deny_list.contains(&address_and_nonce.0) {
```

Comment:
what is the difference between `&address_and_nonce.0` and `tx.initiator_account()` ?

Reply:
there is no difference. I will update the line 42 for consistency.